### PR TITLE
added mdBook docs revision

### PR DESCRIPTION
**Pull request by eldonm**
moved gendocs.py to scripts; updated master README; unlinked and removed LICENSE.md; linked to jaseci_ai_kit READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,279 +1,139 @@
...
+
+## Jaseci Library Reference
+
+- [Jaseci AI Kit](jaseci_ai_kit/README.md)
+    - [CL Summarization](jaseci_ai_kit/jaseci_ai_kit/modules/cl_summer/README.md)
+    - [Bi-Encoder](jaseci_ai_kit/jaseci_ai_kit/modules/encoders/README.md)
+    - [FLair NER](jaseci_ai_kit/jaseci_ai_kit/modules/ent_ext/README.md)
+    - [Fasttext Classifier](jaseci_ai_kit/jaseci_ai_kit/modules/fast_enc/README.md)
+    - [Object Detection](jaseci_ai_kit/jaseci_ai_kit/modules/object_detection/readme.md)
+    - [PDF Extractor](support/guide/jaseci_ai_kit/modules/non_ai/pdf_ext/README.md)
```
Comment (ypkang) [inline README.md]: This is still pointing at support/guide

Comment (ypkang) [reply README.md]: also, i am still seeing duplicate the jaseci_ai_kit readme duplicated exists in support/guide. We don't need those anymore right?

File: support/codelabs/canonicai/main.md [hunk 2]
```diff
@@ -0,0 +1,1589 @@
+# Build a conversational AI system in Jaseci
```
Comment (ypkang) [inline support/codelabs/canonicai/main.md]: This main.md contains all the content from chapter 1- 7md. We can remove chapter 1- 7.

**Review by ypkang — CHANGES_REQUESTED** [review]

**Comment by ypkang** [comment]
I can run the gendocs.py script without errors now. I still don't know how to access the mdbooks instance. This is the output from gendocs. Is there a URL I should be using to access?

```
ypkang@DESKTOP-J0PDH87:~/projects/jaseci-review/jaseci-eldon/scripts$ python gendocs.py
2022-09-28 11:58:30 [INFO] (mdbook::book::init): Creating a new book with stub content

All done, no errors...
2022-09-28 11:58:30 [INFO] (mdbook::book): Book building has started
2022-09-28 11:58:30 [INFO] (mdbook::book): Running the html backend
```

File: README.md (hunk 1 above)
Comment (eldonm) [reply README.md]: PDF Extractor had no readme under jaseci_ai_kit/jaseci_ai_kit so I referenced a version we grabbed earlier. I decided to leave the support/jaseci_ai_kit to guide the folder restructuring for later. There are other markdown files in there which provide descriptions and listings for groups of AI kit items, such as "encoders", etc. I can back it up and remove it altogether if you insist.

File: support/codelabs/canonicai/main.md (hunk 2 above)
Comment (eldonm) [reply support/codelabs/canonicai/main.md]: Noted. Will do

**Comment by eldonm** [comment]
> 

gendocs.py will generate /mdbook, you can run mdbook serve to run it or host the files in the /mdbook/book folder

**Comment by ypkang** [comment]
@marsninja @eldonm I removed the deprecated canonicai codelab from this PR. I was planning to delete the current `docs/` stuff but i don't think every md files in `docs/` today is in the mdbooks hierarchy so I kept those there.

**Pull request merged**